Azure/Azure-Sentinel
# Updating CyberArkEpm Zip

Required items, please complete
   
   Change(s):
   - updating package with latest file

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below
